### PR TITLE
Test legacy e2e flow

### PR DESCRIPTION
**Pull request by mhisltd**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRO-4602

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by douglasrice — CHANGES_REQUESTED** [review]
src/test/end-to-end/paths/legacySearchPath.js

const createCaseConfig = require('src/test/end-to-end/pages/createCase/createCaseConfig.json'); never used so can be removed
I.filterCase(filterCaseConfig.list1_text, filterCaseConfig.list2_text, filterCaseConfig.list3_text); I would remove these parameters and use them within the src/test/end-to-end/pages/filterCase/filterCase.js file itself -> then this line can also be removed: 
const filterCaseConfig = require('src/test/end-to-end/pages/filterCase/filterCaseConfig');


src/test/end-to-end/pages/selectCase/selectCase.js
src/test/end-to-end/pages/search/legacyCaseSearch.js
src/test/end-to-end/pages/search/legacyCaseSearch2.js
src/test/end-to-end/pages/search/legacyCaseSearch3.js
Remove the parameters in the function (for these files) as they are not used

File: src/test/end-to-end/saucelabs.conf.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ const browser = process.env.SAUCELABS_BROWSER || 'chrome';
 const tunnelName = process.env.TUNNEL_IDENTIFIER || '';
 
 const setupConfig = {
-    'tests': './paths/*.js',
+    'tests': './paths/legacySearchPath.js',
```
Comment (br0kenbuild) [inline src/test/end-to-end/saucelabs.conf.js:7]: missed?

File: src/test/end-to-end/pages/search/legacyCaseSearch2.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+//    I.fillField(legacyCaseSearch2Config.deceasedMonthOB, "01");
+//    I.fillField(legacyCaseSearch2Config.deceasedYearOB, "1930");
+//    I.fillField(legacyCaseSearch2Config.deceasedDayOD, "01");
+//    I.fillField(legacyCaseSearch2Config.deceasedMonthOD, "01");
+//    I.fillField(legacyCaseSearch2Config.deceasedYearOD, "2017");
+
+    I.wait(20);
+
+    I.waitForNavigationToComplete(legacyCaseSearch2Config.continueBtn);
+    I.wait(20);
```
Comment (br0kenbuild) [inline src/test/end-to-end/pages/search/legacyCaseSearch2.js]: why? Doug hasn't used it after any of his 18 uses of 'waitForNavigationToComplete'

File: src/test/end-to-end/pages/search/legacyCaseSearch2.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    I.fillField(legacyCaseSearch2Config.forename, "Albert");
+//    I.fillField(legacyCaseSearch2Config.surname, "Surname");
+//    I.fillField(legacyCaseSearch2Config.deceasedDayOB, "01");
+//    I.fillField(legacyCaseSearch2Config.deceasedMonthOB, "01");
+//    I.fillField(legacyCaseSearch2Config.deceasedYearOB, "1930");
+//    I.fillField(legacyCaseSearch2Config.deceasedDayOD, "01");
+//    I.fillField(legacyCaseSearch2Config.deceasedMonthOD, "01");
+//    I.fillField(legacyCaseSearch2Config.deceasedYearOD, "2017");
+
+    I.wait(20);
```
Comment (br0kenbuild) [inline src/test/end-to-end/pages/search/legacyCaseSearch2.js]: why? 'waitForNavigationToComplete' does also:
waitForNavigation({waitUntil: ['domcontentloaded', 'networkidle0']})

File: src/test/end-to-end/pages/search/legacyCaseSearch3.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+module.exports = function (jurisdiction, caseType, event) {
+
+    const I = this;
+    I.waitForText(legacyCaseSearch3Config.waitForText, testConfig.TestTimeToWaitForText);
+    I.seeInField(legacyCaseSearch3Config.legacyCaseType, "Legacy WILL");
+    I.seeInField(legacyCaseSearch3Config.probateManId, "89849");
+    I.seeInField(legacyCaseSearch3Config.fullName, "ALBERT LOWERY");
+    I.click(legacyCaseSearch3Config.yesOption);
+    I.wait(20);
```
Comment (br0kenbuild) [inline src/test/end-to-end/pages/search/legacyCaseSearch3.js]: same here

File: src/test/end-to-end/pages/selectCase/selectCase.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+const testConfig = require('src/test/config.js');
+const selectCaseConfig = require('./selectCaseConfig.json');
+
+module.exports = function (jurisdiction, caseType, event) {
+
+    const I = this;
+    I.waitForText(selectCaseConfig.waitForText, testConfig.TestTimeToWaitForText);
+
+    I.wait(20);
```
Comment (br0kenbuild) [inline src/test/end-to-end/pages/selectCase/selectCase.js]: same here

File: src/test/end-to-end/pages/search/legacyCaseSearch.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+
+const testConfig = require('src/test/config.js');
+const legacyCaseSearchConfig = require('./legacyCaseSearchConfig.json');
+
+module.exports = function (jurisdiction, caseType, event) {
+
+    const I = this;
+    I.waitForText(legacyCaseSearchConfig.waitForText, testConfig.TestTimeToWaitForText);
+
+    I.wait(20);
```
Comment (br0kenbuild) [inline src/test/end-to-end/pages/search/legacyCaseSearch.js]: same here

**Review by br0kenbuild — CHANGES_REQUESTED** [review]

**Pull request closed without merging**